### PR TITLE
Redirects 10.5 Non RAD pages

### PR DESCRIPTION
**Pull request by kathycarothers**
I have put in the redirects for the non RAD pages listed in the ticket.

### CODE REVIEW

File: redirects-transition.conf [hunk 1]
```diff
@@ -1,5 +1,13 @@
 #Redirect rules for specific pages
 
+rewrite ^/eeo/LimitedEnglishProficiencyPlan.shtml https://www.fec.gov/about/equal-employment-opportunity/limited-english-proficiency/ redirect;
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect;
```
Comment (patphongs) [inline redirects-transition.conf]: Redirect URL here should be: https://www.fec.gov/about/equal-employment-opportunity/

File: redirects-transition.conf [hunk 2]
```diff
@@ -1,5 +1,13 @@
 #Redirect rules for specific pages
 
+rewrite ^/eeo/LimitedEnglishProficiencyPlan.shtml https://www.fec.gov/about/equal-employment-opportunity/limited-english-proficiency/ redirect;
+rewrite ^/eeo/eeo.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect; 
+rewrite ^/about/offices/EEO/EEO.shtml https://www.fec.gov/about/equal-employment-opportunity-/ redirect;
```
Comment (patphongs) [inline redirects-transition.conf]: Redirect URL here should be: https://www.fec.gov/about/equal-employment-opportunity/

**Comment by patphongs** [comment]
> Question for @patphongs - #150 and #149 are the same edit, just put in twice. It looks OK but this one is noted "outdated". Should I close this PR without merging it?

@dorothyyeager I think you're saying that the two PRs mentioned above are duplicates to the one that was just merged in https://github.com/fecgov/fec-proxy/pull/151? If that's the case, we can close 150 and 149.

**Comment by dorothyyeager** [comment]
> > Question for @patphongs - #150 and #149 are the same edit, just put in twice. It looks OK but this one is noted "outdated". Should I close this PR without merging it?
> 
> @dorothyyeager I think you're saying that the two PRs mentioned above are duplicates to the one that was just merged in #151? If that's the case, we can close 150 and 149.

No - #151 was for www, 150 and 149 are for transition, just duplicated each other. @patphongs

**Comment by kathycarothers** [comment]
Closing this one since it is a duplicate for 150.

**Pull request merged**